Bexchange.Jwt.API/Controllers/UserController.cs

```diff
@@ -0,0 +1,237 @@
+﻿using BexchangeAPI.Domain.DtoModels;
+using BexchangeAPI.Domain.Enum;
+using BexchangeAPI.Domain.Models;
+using BexchangeAPI.Infrastructure.Repositories.Interfaces;
+using BexchangeAPI.Infrastructure.DtbContext;
+using Microsoft.AspNetCore.Mvc;
+using System.Security.Cryptography;
+using Bexchange.Infrastructure.Repositories.Interfaces;
+using System.Security.Claims;
+using Microsoft.IdentityModel.Tokens;
+using System.IdentityModel.Tokens.Jwt;
+using System.Text;
+using Bexchange.Domain.DtoModels;
+using Microsoft.AspNetCore.Cors;
```

Comment:
Move all additional methods to some service. Controller does not have to contain it